nss-day-cohort-29/reactive-nutshell-copperden
# Austin and Hunter Login

# Description
Login only when a user exists in the database, session storage for both register and login.

## Type of change


# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout ab-login`
3. Check to see that a random user (not in database) cannot login without creating an account.
4. Check in console to see that the user that signed in is printed and set in session storage

# Checklist:
